syndeo-template/src/main/java/com/google/cloud/syndeo/transforms/pubsub/SyndeoPubsubReadSchemaTransformProvider.java

```diff
@@ -209,6 +241,10 @@ public abstract static class SyndeoPubsubReadSchemaTransformConfiguration {
             + "For JSON data, this is a schema defined with JSON-schema syntax (https://json-schema.org/).")
     public abstract String getSchema();
 
+    public abstract @Nullable String getDlqType();
```

Comment:
Do we need to have this and the other one as part of this transform?
I think these should reside with the dlqSchemaTransform as opposed to being declared in every IO schemaTransform.

Reply:
The current PubsubDlqSchemaTransform does have this parameters. But when we are building a Syndeo pipeline in the JsonSpecPaylod we only specify the source and the sink at the top level. We don't add dlq sink as part of the JsonSpec, so in order to get the dlq parameters they need to be passed through the source and sink config parameters. In order to do what you are saying we would have to add dlq as a top level parameter. Something like this:

Old JsonSpecPayload
{
    source
    sink
}


New JsonSpecPayload
{
    source
    sink
    dlq
}

Reply:
This has been done. Now we can add dlq as a top level parameter like this: 

"dlq": {
    "urn": "syndeo:schematransform:com.google.cloud:pubsub_dlq_write:v1",
    "configurationParameters": {
        "topic": "projects/manav-jit-test/topics/dlq-test"
     }
  }